src/Main.cpp

```diff
@@ -393,7 +393,7 @@ void generateDoc(std::string outputDirectory, bool skipEmptyModuleName, const st
                     const auto& absoluteFile = filteredAbsoluteFiles[i];
                     const auto& relativeFile = filteredRelativeFiles[i];
 
-                    ss << relativeFile << "\n\n";
+                    ss << "[" << relativeFile << "](https://github.com/sofa-framework/sofa/blob/master/examples/" << relativeFile << ")" << "\n\n";
```

Comment:
Currently, the plugins are also parsed, including the external ones. So this url won't work for them